compiler/luci/export/src/CircleOperationExporter.cpp

```diff
@@ -1724,17 +1726,53 @@ void exportNode(loco::Node *node, flatbuffers::FlatBufferBuilder &builder, Seria
 }
 
 } // namespace
+#endif
 
 namespace luci
 {
 
-void exportNodes(loco::Graph *g, FlatBufferBuilder &builder, SerializedModelData &md,
+void exportNodes(loco::Graph *g, flatbuffers::FlatBufferBuilder &builder, SerializedModelData &md,
                  SerializedGraphData &gd)
 {
   uint32_t node_position = 0;
   for (auto node : loco::postorder_traversal(loco::output_nodes(g)))
   {
-    exportNode(node, builder, md, gd, node_position);
+    // exportNode(node, builder, md, gd, node_position);
+    if (auto circle_node = dynamic_cast<luci::CircleNode *>(node))
```

Comment:
as all nodes should be circle nodes, we may use `must_cast`

Reply:
I just copied original code without change and following `else` statement is found.

```cpp
    else
    {
      INTERNAL_EXN("Node with unsupported dialect found");
    }
```

Would it be better to change to `must_cast` and remove if-else statement? :D
(I think using `must_cast` is much pretty :D)

Reply:
> Would it be better to change to must_cast and remove if-else statement? :D

Yes, you can go as-is for this PR and change later or apply in this PR.